kolibri/plugins/device/assets/src/views/lodUsers/importUser/ImportUserWithCredentialsPage.vue

```diff
@@ -0,0 +1,403 @@
+<template>
```

Comment:
When first implemented the issue some months ago, we decided to duplicate these components from the setup wizard https://github.com/AlexVelezLl/kolibri/blob/87eb263c135bb4f0e629035a9a2bece43dc52483/kolibri/plugins/setup_wizard/assets/src/views/ImportIndividualUserForm.vue#L25. Do we still want to keep these components duplicated? Or should we find a way to encapsulate the common logic in just one component in the scope of this PR?

Reply:
I'm not bothered by the duplication between the two here for now. If we anticipate that this workflow will change sooner than later and it'd be nice to have cleaner code to work with when that time comes, then it'd probably be worth making it a priority to fix in a follow-up.

Unless there is some reason I'm not seeing beyond generally avoiding duplication where we can, then I think it'd be okay as a clean-up task.

Reply:
I agree - I think the duplication is okay for now. @AlexVelezLl and I chatted about this briefly during our checkin today and my two justifications are: 1) it's only in two places, let's abstract at 3 places. 2) I think the benefits of having this merged in so we can have it in `develop` as part of the 0.19 work (for QA/testing edge cases, etc) outweigh doing some refactoring now.